src/mocks/generate-mock-documents.js

```diff
@@ -0,0 +1,155 @@
+const { writeFileSync } = require('fs')
+const { getStudents, getStudent, getStudentClasses, getStudentTeachers } = require('./pifu')
+const { DOCUMENTS: { behaviourReasons, courseReasons, orderReasons, periods } } = require('../data/documents')
+const documentsToCreate = 80
+
+const { data: students } = getStudents()
+const documents = []
+
+const randomId = () => {
+  const timestamp = (new Date().getTime() / 1000 | 0).toString(16)
+  return timestamp + 'xxxxxxxxxxxxxxxx'.replace(/[x]/g, () => {
+    return (Math.random() * 16 | 0).toString(16)
+  }).toLowerCase()
+}
+
+const getRandomDate = () => {
+  const start = new Date(2020, 0, 1)
+  const end = new Date(2021, 6, 15)
+  const date = new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()))
+  if (date.getDay() === 0 || date.getDay() > 5) date.setDate(date.getDate() + 3)
+  if (date.getHours() < 7 || date.getHours() > 18) date.setHours(Math.floor(Math.random() * (18 - 7 + 1)) + 7)
+  return date
+}
+
+const getRandomVariant = () => {
+  const variants = [
+    { type: 'varsel', variant: 'orden' },
+    { type: 'varsel', variant: 'orden' },
+    { type: 'varsel', variant: 'orden' },
+    { type: 'varsel', variant: 'atferd' },
+    { type: 'varsel', variant: 'atferd' },
+    { type: 'varsel', variant: 'atferd' },
+    { type: 'varsel', variant: 'fag' },
+    { type: 'varsel', variant: 'fag' },
+    { type: 'varsel', variant: 'fag' },
+    { type: 'varsel', variant: 'fag' },
+    { type: 'varsel', variant: 'fag' },
+    { type: 'varsel', variant: 'fag' },
+    { type: 'samtale', variant: 'samtale' },
+    { type: 'samtale', variant: 'ikke-samtale' },
+    { type: 'notat', variant: 'notat' },
+    { type: 'notat', variant: 'notat' },
+    { type: 'notat', variant: 'notat' },
+    { type: 'notat', variant: 'notat' },
+    { type: 'notat', variant: 'notat' }
+  ]
```

Comment:
Ser ikke pent ut, men er gjort sånn for å få riktig fordeling av typer (så det kommer flere varsler i fag, enn elevsamtaler feks)